### ISSUE TITLE
open for maintainers!

### ISSUE THREAD

**Issue by ssbarnea** [pycontribs/xmlrunner#10]
If anyone wants to takeover maintenance of this project please add yourself here. I no longer have the time and interest for it and I will be glad to allow someone to takeover.

**Comment by rasjani** [comment]
Whats the difference between this and original fork  as it seems to be once again maintained ?

**Comment by ssbarnea** [comment]
@rasjani Can you provide some links?

I took this project a very long time ago and I do not remember all the details but I am pretty sure it was in order to fix xmlrunner which was broken and unmaintained.

AFAIK, https://pypi.org/project/xmlrunner/#history has last release in 2014.

**Comment by rasjani** [comment]
If you go to this github.com projects main page, it says that this repo is a fork of "https://github.com/xmlrunner/unittest-xml-reporting", which indeed looks very similar and that repo seems to be maintained - atleast there are lots of commits recently.

**Comment by ssbarnea** [comment]
Indeed! We will have to check which other packages are listing xmlrunner as dependency and if they break with the original package. 

If no problems are encountered I will archive this fork and even handle the xmlrunner package to uittest-xml-reporting if they are willing to publish under this name, which matches the package name and the CLI ... making is less confusing for the users.
